src/dom/context2d.js

```diff
@@ -1,22 +1,20 @@
 export default function(width, height, options) {
-  var colorSpace = "srgb";
   var scale = devicePixelRatio;
-  if (options == +options) {
+  if (options == null) {
+    options = {};
+  } else if (options == +options) {
     scale = +options;
+    options = {};
   } else {
     if (options.scale) scale = +options.scale;
-    if (options.colorSpace === "display-p3") colorSpace = options.colorSpace;
+    delete options.scale;
```

Comment:
Oops, this mutates the user’s passed-in object! You’ll want to use the rest operator instead.

```js
const {scale, ...rest} = options;
options = rest;
```